src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -146,15 +150,32 @@ class Monitoring final
                                         }
                                         else
                                         {
+                                            logWarn(IC_MONITORING,
+                                                    "Cluster health status: '%s'",
+                                                    Utils::toUpperCase(fields.at(HealthCheckColumns::STATUS)).c_str());
                                             value.second = false;
                                         }
                                     }
                                     else
                                     {
+                                        logDebug2(IC_MONITORING,
```

Comment:
Improve: For the same reasons detailed above, I'd add the name of the server that failed.

Reply:
done